streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamSlidingWindowAggregateTest.java

```diff
@@ -92,6 +94,7 @@
 import static org.junit.Assert.assertTrue;
 
 @RunWith(Parameterized.class)
+@Category({IntegrationTest.class})
```

Comment:
Looking into gradle enterprise, I cannot find any failing test run in the last 90 days for this test...

Reply:
The average duration for this test is ~7m30s, the 24th longest test suite in the project.